pkg/components/util/delete.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	"helm.sh/helm/v3/pkg/action"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kinvolk/lokomotive/pkg/components"
+	"github.com/kinvolk/lokomotive/pkg/k8sutil"
+)
+
+// DeleteComponent deletes a component.
+func DeleteComponent(name string, c components.Component, kubeconfig string, deleteNS bool) error {
```

Comment:
I think we should avoid using the "utility package" pattern as much as we can. This has a broader context than just this PR so I've opened https://github.com/kinvolk/lokomotive/issues/290 to track this.

In this specific instance, IMO `DeleteComponent()` should reside directly under `pkg/components`.

Reply:
> In this specific instance, IMO `DeleteComponent()` should reside directly under `pkg/components`.

I agree, I don't think we need this `util` package.